frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationItem.tsx

```diff
@@ -95,18 +106,41 @@ export const MultilineUtilizationItem: React.FC<MultilineUtilizationItemProps> =
     });
 
     return (
-      <div className="co-utilization-card__item" data-test-id="utilization-item">
-        <div className="co-utilization-card__item-description">
+      <div
+        className={isOCSRendered ? 'co-utilization-card__item-ceph' : 'co-utilization-card__item'}
```

Comment:
Remove all instances of `isOCSRendered` we cannot mention ocs in common code. Instead introduce props for properties you want to enhance.